app/controllers/products_common_controller.rb

```diff
@@ -157,12 +152,28 @@ def translation_scope
 
   private
 
+  def create_params
+    params.require(:"#{singular_object_name}").permit(:name, :url_name, :contact_email, :description,
+                                                      :facility_account_id, :account, :initial_order_status_id,
+                                                      :requires_approval, :training_request_contacts,
+                                                      :is_archived, :is_hidden, :order_notification_recipient,
+                                                      :user_notes_field_mode, :user_notes_label, :show_details,
+                                                      :schedule_id, :control_mechanism, :reserve_interval,
+                                                      :min_reserve_mins, :max_reserve_mins, :min_cancel_hours,
+                                                      :auto_cancel_mins, :lock_window, :cutoff_hours,
+                                                      relay_attributes: [:ip, :port, :username, :password, :type, :instrument_id])
+  end
+
+  def update_params
```

Comment:
Did we figure out you could use something like `resource_params` and have cancan use it for both create and update? Not sure if it's worth changing that, though.